Furyfile.yml

```diff
@@ -10,8 +9,45 @@ roles:
 modules:
   - name: aws/aws-vpc
   - name: aws/aws-kubernetes
+  - name: terraform-aws-acm
+    version: v2.5.0
+    provider:
+      name: aws
+      label: ufficial-modules
+    registry: true
+  - name: tf_aws_bastion_s3_keys
+    version: v2.0.0
+    provider:
+      name: aws
+      label: community-modules
+    registry: true
+  - name: terraform-google-kubernetes-engine
+    version: v6.2.0
+    provider:
+      name: gcp
+      label: ufficial-modules
+    registry: true
+  - name: terraform-azurerm-aks
+    version: v2.0
+    provider:
+      name: azure
+      label: ufficial-modules
+    registry: true
 
 bases:
   - name: monitoring
   - name: logging
     version: master
+
+tfRepos:
```

Comment:
What about changing `tfRepos` to `provider.module`?

I mean, what if tomorrow we need to do the same with roles or katalog (kustomize)?

The change will looks like:

Before:
```yaml
tfRepos:
  aws:
    - url: https://github.com/terraform-aws-modules
      label: ufficial-modules
```

To

```yaml
provider:
  modules:
    aws:
      - url: https://github.com/terraform-aws-modules
        label: ufficial-modules
  roles: {} #TBD
  katalog: {} #TBD
```

Reply:
could be an idea, but is better to implement it as asoon as we will have the real need (the same that push me to deepen this development). The risk of the overengineer is to build something that won't match the need, once the need will appear. Just to explain it better: try to consolidate upfront can expose a wrong abstraction of the other 2 (roles and katalog) about what now we have no information on how could be a future implementation.

Reply:
i applied the change you've asked, give a look 👀

Reply:
Now looks very good!!! Thanks @lzecca78 !